src/strings.cpp

```diff
@@ -1627,6 +1627,12 @@ static void FormatString(StringBuilder &builder, const char *str_arg, StringPara
 				break;
 			}
 
+			case SCC_COLOUR: { // {COLOUR}
+				StringControlCode scc = (StringControlCode)(SCC_BLUE + args.GetInt32(SCC_COLOUR));
```

Comment:
I'm getting a compile error on this line after the merge.
```
C:\Users\nielsm\Dev\OpenTTD\src\strings.cpp(1628): error C2660: 'StringParameters::GetInt32': function does not take 1 arguments
  C:\Users\nielsm\Dev\OpenTTD\src\strings_internal.h(91): note: see declaration of 'StringParameters::GetInt32'
  C:\Users\nielsm\Dev\OpenTTD\src\strings.cpp(1628): note: while trying to match the argument list '(StringControlCode)'
```

As far as I can tell, `StringParameters::GetInt32()` has never existed in a version that takes a parameter.